huaweicloud/services/evs/resource_huaweicloud_evs_volume.go

```diff
@@ -181,6 +193,15 @@ func resourceVolumeAttachmentHash(v interface{}) int {
 	return hashcode.String(buf.String())
 }
 
+func validatePrePaidParameter(d *schema.ResourceData) error {
```

Comment:
we can extend this function without **PrePaid**.

```suggestion
func validateParameter(d *schema.ResourceData) error {
```

Reply:
OK